ecc_enclave/enclave/enclave.cpp

```diff
@@ -134,6 +154,26 @@ int ecall_cc_invoke(const uint8_t* signed_proposal_proto_bytes,
             crm.has_proposal = true;
         }
 
+        {  // fill chaincode request message hash
+            // hash request
+            ByteArray ba_cc_request_message(
+                cc_request_message_bytes, cc_request_message_bytes + cc_request_message_bytes_len);
+            ByteArray ba_cc_request_message_hash =
+                pdo::crypto::ComputeMessageHash(ba_cc_request_message);
```

Comment:
~this hash looks rather redundant?~

Oh, forgot that we do also send this as part of message to make protobuf easier (even though that of course is also redunant. So hash keeps message smaller. ..